CODEOWNERS

```diff
@@ -86,6 +86,7 @@ homeassistant/components/coronavirus/* @home_assistant/core
 homeassistant/components/counter/* @fabaff
 homeassistant/components/cover/* @home-assistant/core
 homeassistant/components/cpuspeed/* @fabaff
+homeassistant/components/crownstone/* @Crownstone @RicArch97
```

Comment:
Are you affiliated with Crownstone?

Are they aware they are set as a code owner?

Reply:
I started developing this integration as part of an internship for Crownstone. After it ended I decided to keep developing and maintaining the integration, but not as official member. This means Crownstone is the official owner of it, and they will take over maintenance in case I am unavailable.

Reply:
We are fine with it. ~CEO Crownstone.